tests/webdriver-e2e/page-objects/login.page.js

```diff
@@ -0,0 +1,47 @@
+const utils = require('../../utils');
+
+const login = async (username, password) => {
+  await (await this.inputUsername).setValue(username);
+  await (await this.inputPassword).setValue(password);
+  await (await this.btnSubmit).click();
+};
+
+const cookieLogin = async (username, password) => {
+  const opts = {
+    path: '/medic/login',
+    body: { user: username, password: password },
+    port: 5988,
+    method: 'POST',
+    simple: false,
+    headers: {
+      'Content-Type': 'application/json'
+    }
+  };
+  const resp = await utils.request(opts);
+  const cookieArray = [];
+  let cookie = resp.headers['set-cookie'].find(x => x.includes('Auth')).split(';');
```

Comment:
I think this code of extracting cookies from response headers should be part of utils and return an array or a "dictionary" with set cookies, that we could reuse.

Reply:
agreed!